docs/pages/includes/config-reference/auth-service.yaml

```diff
@@ -1,465 +1,464 @@
-       - tls-ecdhe-rsa-with-chacha20-poly1305
-       - tls-ecdhe-ecdsa-with-chacha20-poly1305
+  # Configuration for the storage back-end used for the cluster state and the
+  # audit log. Several back-end types are supported.
+  storage:
+    # By default teleport uses a SQLite database in the `data_dir`
+    # directory on a local filesystem
+    type: sqlite
+
+    # List of locations where the audit log events will be stored. By
+    # default, they are stored in `/var/lib/teleport/log`.
+    #
+    # When specifying multiple destinations like this, make sure that
+    # highly-available storage methods (like DynamoDB or Firestore) are
+    # specified first, as this is what the Teleport Web UI uses as its
+    # source of events to display.
+    audit_events_uri:
+      - "dynamodb://events_table_name"
+      - "firestore://events_table_name"
+      - "postgresql://user_name@database-address/events_table_name"
+      - "file:///var/lib/teleport/log"
+      - "stdout://"
+
+    # Use this setting to configure teleport to store the recorded sessions
+    # in an AWS S3 bucket or use GCP Storage with 'gs://'.
+    audit_sessions_uri: "s3://example.com/path/to/bucket?region=us-east-1"
+
+    # SQLite-specific section:
+
+    # The default path is the `backend` directory in the `data_dir`
+    path: /var/lib/teleport/backend/
+    # SQLite's `synchronous` pragma, can be set to `"OFF"` for improved
+    # write performance in exchange for reliability against system crashes
+    # (see https://www.sqlite.org/pragma.html#pragma_synchronous).
+    sync: FULL
+    # SQLite's `journal_mode` pragma, by default it doesn't change the mode from
+    # the SQLite default (DELETE unless the database file is using WAL mode).
+    # For improved performance without sacrificing reliability it's possible to
+    # set `journal` to `WAL` and `sync` to `NORMAL`, but only when using a filesystem
+    # that supports locks (see https://www.sqlite.org/pragma.html#pragma_journal_mode).
+    #journal: DELETE
+
+    # DynamoDB-specific section:
+
+    # continuous_backups is used to enable continuous backups.
+    # default: false
+    continuous_backups: true
+
+    # auto_scaling is used to enable (and define settings for) auto
+    # scaling.
+    # default: false
+    auto_scaling: true
+
+    # By default, Teleport stores stores audit events with an AWS TTL of 1 year.
+    # This value can be configured as shown below. If set to 0 seconds, TTL is disabled.
+    #
+    # NOTE: Only the DynamoDB events backend respects the retention_period. All other event backends
+    # consume the retention period via a query parameter in the audit_events_uri. See the examples below
+    # for how to configure the retention period for other backends.
+    # Firestore: firestore://events_table_name?eventRetentionPeriod=10d
+    # Postgres: postgresql://user_name@database-address/teleport_audit?sslmode=verfify-full#retention_period=240h
```

Comment:
This is the only line changed in the PR.

Apparently the indentation on this entire file was off :-(